sql/moz-fx-data-shared-prod/fenix/retention_clients/view.sql

```diff
@@ -0,0 +1,78 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.fenix.retention_clients`
+AS
+WITH clients_last_seen AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+    normalized_channel,
+    mozfun.bits28.retention(days_seen_bits, submission_date) AS retention_seen,
+    mozfun.bits28.retention(days_active_bits & days_seen_bits, submission_date) AS retention_active,
+    days_seen_bits,
+    days_active_bits,
+  FROM
+    `moz-fx-data-shared-prod.fenix.baseline_clients_last_seen`
+),
+attribution AS (
+  SELECT
+    client_id,
+    sample_id,
+    channel AS normalized_channel,
+    adjust_ad_group,
+    adjust_campaign,
+    adjust_creative,
+    adjust_network,
+    install_source,
```

Comment:
I received a request to have all the different stores on this field. As it is right now, firefox_android_clients is grouping values as "Other"
`fenix_derived.firefox_android_clients_v1` contains the info that we need.